main.tf

```diff
@@ -379,7 +379,8 @@ module "efs_sg" {
   vpc_id      = local.vpc_id
   description = "Security group allowing access to the EFS storage"
 
-  ingress_cidr_blocks = [var.cidr]
+  auto_ingress_rules = []
```

Comment:
Setting it like this disables the main purpose of this security group submodule which creates a predefined rule (for `nfs` in this case):

https://github.com/terraform-aws-modules/terraform-aws-security-group/blob/master/modules/nfs/auto_values.tf#L6-L10